examples/client_server/README.md

```diff
@@ -0,0 +1,27 @@
+# Client Server Demo
+
+A very simple example to show peer to peer communication with rpc calls.

```

Comment:
RPC is not a Godot specific thing, so it might be good to clarify that this is using Godot's high-level multiplayer API here.